src/libstd/panicking.rs

```diff
@@ -221,7 +221,7 @@ impl<'a> PanicInfo<'a> {
     ///
     /// panic::set_hook(Box::new(|panic_info| {
     ///     if let Some(location) = panic_info.location() {
-    ///         println!("panic occurred in file '{}' at line {}", location.file(), location.line());
+    ///        println!("panic occurred in file '{}' at line {}", location.file(), location.line());
```

Comment:
By convention, Rust uses four spaces for indentation, you'll need to do something like this instead:

```rust
    ///        println!("panic occurred in file '{}' at line {}", 
    ///                 location.file(), location.line());
````